Docs/source/Hardware/Boards.rst

```diff
@@ -68,23 +68,115 @@ A switching regulator (NCP1117) converts the +5V input to +3.3V. This does not n
+           | forced ventilation system. Such ventilation  | system and in the event of a battery protection 
+           | systems must pull exhaust to the exterior of | fault it's powered by the supplemental battery.
+           | the solar car and must be directly connected 
+           | to the exterior of the vehicle away from any 
+           | airstream that may reach the driver. The 
+           | ventilation system shall be powered by the 
+           | battery system. In the event of a Battery 
+           | Protection Fault, provisions should be made 
+           | to power this fan from the Supplemental 
+           | battery.
+========== ============================================== ===============================================
+
+Context
+^^^^^^^
+**Location of the Board:** With the BPS in the battery box 
+
+List of I/O and Connections: 
+    
+    * Power +12 V 
+        * Input from BPS Leader Board 
+    * Power GNDPWR 
+        * Input from BPS Leader Board 
+    * FAN 1 
+        * :term:`PWM <Pulse Width Modulation>` Input from BPS Leader Board 
+    * FAN 2 
+        * :term:`PWM <Pulse Width Modulation>` Input from BPS Leader Board 
+    * FAN 3 
+        * :term:`PWM <Pulse Width Modulation>` Input from BPS Leader Board 
+    * Fan 4 
+        * :term:`PWM <Pulse Width Modulation>` Input from BPS Leader Board 
+
+Schematic
+^^^^^^^^^
+*What does this circuit do?*
+    They control the fan's speed for cooling the battery pack. 
+*Why do we need it?*
+    This board makes sure the batteries don’t heat up by controlling the speed of the fans based on
+    the temperature. 
+*List of Circuit Components*
+    * Connector_Molex:Molex_Micro-Fit_3.0_43045-0612_2x03_P3.00mm_Vertical (LDRBRDConn) 
+        * Description: connects the leader board to fan board 
+        * Why is it necessary: so the fans can be supplied power 
+        * Justification for selection of specific part: this connector provides the right amount 
+          of power and connections for all fans being used 
+        * Associated passives/components:  
+            * 4 different BUK9M34-100EX 
+    * BUK9M34-100EX(4) 
+        * Description: a :term:`MOSFET <MOSFET>` that will switch the fans off and on 
+        * Why is it necessary: so the fans can be set to certain speeds by turning them on and off at varying rates
+        * Justification for selection of specific part: this part is used over others because of 
+          Q101 compliant, its suitable for thermally demanding environments, and true logic gate 
+          with VGS(th) rating of greather than 0.5V at 175°C 
+        * `Datasheet <https://assets.nexperia.com/documents/data-sheet/BUK9M34-100E.pdf>`__ 
+        * Associated passives/components:  
+            * 4 different diodes, Molex_MicroFit3.0_1x2xP3.00mm_PolarizingPeg_Vertical and the 
+              LDRBDConn above. 
+
+*List of Subsheet I/O*
```

Comment:
Is this supposed to be bold? It's not showing up as bold for me when I build on my machine

Reply:
I made it bold, it was italicized